### PR TITLE
CI - CodeQL Jenkins job

### PR DESCRIPTION
**Pull request by rodrigonull**
ISSUE: https://github.com/apache/incubator-kie-issues/issues/610

### CODE REVIEW

**Comment by jomarko** [comment]
I think similar PRs are useful. Codeql is new for me. I was able to see results of scan for example here https://github.com/apache/incubator-kie-tools-temporary-rnd-do-not-use/actions/runs/6561589447/job/17823110606?pr=9 , however I do not see for example a severity of particular issues, is that possible to find somewhere?

**Comment by rodrigonull** [comment]
@jomarko I'm also new to codeQL and AFAIK you need write permission in the repo to be able to see the detailed information about the code scanning. More info here: https://docs.github.com/en/code-security/code-scanning/managing-code-scanning-alerts/triaging-code-scanning-alerts-in-pull-requests.

File: .ci/jenkins/Jenkinsfile.codeql [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        timeout(time: 30, unit: 'MINUTES')
+    }
+
+    environment {
+        RESULTS_DIR = "/tmp/codeql/results"
+        DATABASE_DIR = "/tmp/codeql/db"
+        GO_RESULTS_DIR = "${RESULTS_DIR}/go"
+        GO_DATABASE_DIR = "${DATABASE_DIR}/go"
+        JS_RESULTS_DIR = "${RESULTS_DIR}/js"
+        JS_DATABASE_DIR = "${DATABASE_DIR}/js"
```
Comment (jomarko) [inline .ci/jenkins/Jenkinsfile.codeql:37]: There is also java code https://github.com/kiegroup/kie-tools/tree/main/packages/extended-services-java

are we going to scan it too?

**Review by jomarko — COMMENTED** [review]
one question, thank you for a PR

File: .ci/jenkins/Jenkinsfile.codeql (hunk 1 above)
Comment (rodrigonull) [reply .ci/jenkins/Jenkinsfile.codeql]: Currently the workflow we have running in the original KIE tools repo only scans JS and GO code, but we can definitely add the scan for Java code too as part of this migration.

**Comment by rodrigonull** [comment]
Closing it in favor of https://github.com/apache/incubator-kie-tools-temporary-rnd-do-not-use/pull/19

**Pull request closed without merging**